packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.tsx

```diff
@@ -21,11 +21,16 @@ import { CustomPhraseResponse } from './types';
 type LanguageEditorProps = {
   selectedLanguageTag: LanguageTag;
   onFormStateChange: (isDirty: boolean) => void;
+  onUpdated: (data: CustomPhraseResponse) => void;
 };
 
 const emptyUiTranslation = createEmptyUiTranslation();
 
-const LanguageEditor = ({ selectedLanguageTag, onFormStateChange }: LanguageEditorProps) => {
+const LanguageEditor = ({
+  selectedLanguageTag,
+  onFormStateChange,
+  onUpdated,
```

Comment:
what would be the difference between onFormStateChange and onUpdate? Looks a little bit confusing.

Reply:
`onFromStateChange` mentions the dirty state of the `LanguageEditor`, and `onUpdated` is a callback that tells the language list a new custom language has been added.

Reply:
The naming may be confusing, thinking about it.

Reply:
updated with `onFormDirtyStateChange` and `onCustomPhraseUpserted`.